iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/helpers/BasicTierOnlyRule.java

```diff
@@ -1,16 +0,0 @@
-/*
- *  Copyright (c) Microsoft. All rights reserved.
- *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
- */
-
-package com.microsoft.azure.sdk.iot.common.helpers;
-
-public class BasicTierOnlyRule implements ConditionalIgnoreRule.IgnoreCondition
-{
-
-    @Override
```

Comment:
The concept of this rule still exists, but I renamed it and refactored it to match the other tags